packages/utils/package.json

```diff
@@ -49,6 +49,7 @@
   "devDependencies": {
     "@types/js-yaml": "^4.0.5",
     "@types/triple-beam": "^1.3.2",
+    "@types/yargs": "^17.0.24",
```

Comment:
I added yargs types as dev dependency to the package as I want to avoid having this as a dependency but this might lead to the case where yargs types result in `any` if the consuming project does not install them.

e.g. something like this would not throw an type error

```ts
registerCommandToYargs({}, { command: "", describe: "" });
```

but if you install `@types/yargs` as dependency in that project it will complain that first param is not correct

```ts
Argument of type '{}' is not assignable to parameter of type 'Argv<{}>'.
  Type '{}' is missing the following properties from type 'Argv<{}>': alias, argv, array, boolean, and 66 more.ts(2345
```

Since this is just an edge use case and all our packages have the types installed, I opted for not adding types as a dependency.

Reply:
Using a `devDependency` in the `src` does not seems reasonable. We need to look for the solution to this type issue.

Reply:
It's find to add types as dev dependency. If we are reexporting the types from this package then we may try duplicating those for now in our source.

Reply:
I don't think duplicating types for this use case makes sense, and consumers can always install the types if they want to.